source/includes/_account.md

```diff
@@ -963,12 +962,10 @@ accountApi.get_emails(account_id, api_key, api_secret)
 > Example Response:
 
 ```json
-[
-   {
-      "accountId":"e4ca38b3-934d-42e8-a292-ffb0af5549f2",
-      "email":"email@laposte.com"
-   }
-]
+{
```

Comment:
The response is an array, I think you accidentally deleted the `[`/`]` from the response?

Reply:
![image](https://github.com/killbill/slate/assets/75751774/17226621-232b-4e98-8936-07700fe10fbc)

Hi Reshma, it is already an array in my local.